docs/root/intro/life_of_a_request.rst

```diff
@@ -0,0 +1,647 @@
+  their ingress listeners.
+
+.. image:: /_static/lor-topology-service-mesh.svg
+   :width: 80%
+   :align: center
+
+.. image:: /_static/lor-topology-service-mesh-node.svg
+   :width: 40%
+   :align: center
+
+
+Envoy is used in a variety of configurations beyond the service mesh. For example, it can also act
+as an internal load balancer:
+
+.. image:: /_static/lor-topology-ilb.svg
+   :width: 65%
+   :align: center
+
+Or as an ingress/egress proxy on the network edge:
+
+.. image:: /_static/lor-topology-edge.svg
+   :width: 90%
+   :align: center
+
+In practice, a hybrid of these is often used, where Envoy features in a service mesh, on the edge
+and as an internal load balancer. A request path may traverse multiple Envoys.
+
+.. image:: /_static/lor-topology-hybrid.svg
+   :width: 90%
+   :align: center
+
+Envoy may be configured in multi-tier topologies for scalability and reliability, with a request
+first passing through an edge Envoy prior to passing through a second Envoy tier:
+
+.. image:: /_static/lor-topology-tiered.svg
+   :width: 80%
+   :align: center
+
+In all the above cases, a request will arrive at a specific Envoy via TCP, UDP or Unix Domain
+Sockets from downstream. Envoy will forward requests upstream via TCP, UDP or Unix Domain Sockets.
+We focus on a single Envoy proxy below.
+
+Configuration
+-------------
+
+Envoy is a very extensible platform. This results in a combinatorial explosion of possible request
+paths, depending on:
+
+* L3/4 protocol, e.g. TCP, UDP, Unix Domain Sockets.
+* L7 protocol, e.g. HTTP/1, HTTP/2, HTTP/3, gRPC, Thrift, Dubbo, Kafka, Redis and various databases.
+* Transport socket, e.g. plain text, TLS, ALTS.
+* Connection routing, e.g. PROXY protocol, original destination, dynamic forwarding.
+* Authentication and authorization.
+* Circuit breakers and outlier detection configuration and activation state.
+* Many other configurations for networking, HTTP, listener, access logging, health checking, tracing
+  and stats extensions.
+
+It's helpful to focus on one at a time, so this example covers the following:
+
+* An HTTP/2 request with :ref:`TLS <arch_overview_ssl>` over a TCP connection for both downstream
+  and upstream.
+* The :ref:`HTTP connection manager <arch_overview_http_conn_man>` as the only :ref:`network filter
+  <arch_overview_network_filters>`.
+* A hypothetical CustomFilter and the `router <arch_overview_http_routing>` filter as the :ref:`HTTP
+  filter <arch_overview_http_filters>` chain.
+* :ref:`Filesystem access logging <arch_overview_access_logs_sinks>`.
+* :ref:`Statsd sink <envoy_v3_api_msg_config.metrics.v3.StatsSink>`.
+* A single :ref:`cluster <arch_overview_cluster_manager>` with static endpoints.
+
+We assume a static bootstrap configuration file for simplicity:
+
+.. code-block:: yaml
+
+  static_resources:
+    listeners:
+    # We have a single listener bound to port 443.
```

Comment:
optional nit 
We have a -> There is a
for consistency with the rest below

Also love the comments here.  So helpful